code/game/objects/items/weapons/teleportation.dm

```diff
@@ -151,7 +151,7 @@ Frequency:
 	var/list/L = list(  )
 	for(var/obj/machinery/computer/teleporter/R in machines)
 		for(var/obj/machinery/teleport/hub/com in locate(R.x + 2, R.y, R.z))
-			if(R.locked && !R.one_time_use)
+			if(R.locked)
```

Comment:
One time use is the thing that's a fragment from the clown planet card that
sent you to the clown planet on centcomm.